static/panes/compiler.ts

```diff
@@ -118,7 +121,7 @@ type LinkedCode = {
     };
 };
 
-type Decorations = Record<string, monaco.editor.IModelDeltaDecoration[] | monaco.editor.IModelDeltaDecoration>;
+type Decorations = Record<string, monaco.editor.IModelDeltaDecoration[]>;
```

Comment:
I made this an array of declaration deltas everywhere, it simplifies some later code